Orange/widgets/data/owtable.py

```diff
@@ -491,6 +485,7 @@ def sizeHint():
         return QSize(800, 500)
 
     @Inputs.data
+    @check_sql_input
```

Comment:
I think this decorator was left out intentionally. 
The widget handles big sql tables differently then other widget.